File: 02Algorithm_Study/01_week_homeWork/K_order.py
def solution(array, commands):
    answer = []
    for i in range(len(commands)):
        startIn = commands[i][0] - 1
        endIn = commands[i][1]
        tmpList = array[startIn:endIn]
        tmpList = sorted(tmpList)
        returnIn = commands[i][2] - 1
        answer.append(tmpList[returnIn])
    return answer

File: 02Algorithm_Study/01_week_homeWork/K_order.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    int count = commands.size();
    
    for(int i = 0; i < count; i ++){
        vector <int> vecTmp;
        for(int j = commands[i][0] - 1; j < commands[i][1]; j ++){
            int tmp = array[j];
            vecTmp.push_back(tmp);
        }
        sort(vecTmp.begin(),vecTmp.end());
        answer.push_back(vecTmp[commands[i][2] - 1]);
    }
    
    return answer;
}
